rules/xcodeproj.bzl

```diff
@@ -693,8 +693,11 @@ def _add_pre_post_actions(target_name, scheme, key, actions):
         actions_to_add = actions[action_type]
         payload = scheme[action_type]
         list = []
+
+        # Note `settingsTarget` is explicitly omitted because
+        # for a target with lots of env vars we might exceed Xcode arg_max limit
         for action in actions_to_add:
-            list.append({"script": action, "settingsTarget": target_name})
+            list.append({"script": action})
```

Comment:
Can you clarify the impact of this change? This is a recent change so I doubt many people are using it, just wanted to understand how this limits the script.